Backend/dev/migrations/rider-app/1149-add-new-messages-for-yatri-sathi.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE atlas_app.merchant_service_usage_config ADD COLUMN enable_dashboard_sms Boolean;
```

Comment:
Here it should be the ServiceProviderName instead of boolean, right?

Reply:
No, it is a boolean variable to store whether the the current merchant is enabled to send dashboard sms or not. It's in the rider-app side. The same is done in the dynamic-offer-driver-side but in transporter config